.ci/docker/common/install_conda.sh

```diff
@@ -51,12 +64,22 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   as_jenkins conda create -n py_$ANACONDA_PYTHON_VERSION -y python="$ANACONDA_PYTHON_VERSION"
 
   # Install PyTorch conda deps, as per https://github.com/pytorch/pytorch README
-  CONDA_COMMON_DEPS="astunparse pyyaml mkl=2021.4.0 mkl-include=2021.4.0 setuptools"
-  if [ "$ANACONDA_PYTHON_VERSION" = "3.11" ] || [ "$ANACONDA_PYTHON_VERSION" = "3.12" ]; then
-    conda_install numpy=1.26.0 ${CONDA_COMMON_DEPS}
+  if [[ $(uname -m) == "aarch64" ]]; then
+    CONDA_COMMON_DEPS="astunparse pyyaml setuptools patchelf==0.17.2 pygit2==1.13.2 openblas==0.3.25=*openmp* ninja==1.11.1 scons==4.5.2"
```

Comment:
Why do we need pygit2 or patchelf to build?
```suggestion
    CONDA_COMMON_DEPS="astunparse pyyaml setuptools openblas==0.3.25=*openmp* ninja==1.11.1 scons==4.5.2"
```

Reply:
probably not because we are not packaging the wheels for external use. I will clean it up

Reply:
removed pygit2 and patchelf.